docs/github/v2/15-deployment/ios.md

```diff
@@ -1,45 +1,46 @@
-# Publish to AppStore
+# Release to the App Store
```

Comment:
So far we have been using `publish` as the word to imply that a `build` for a `release` is going to the platform where it will be made available. We could also use the word `deploy` instead, but I'd say `release` would be a bit ambiguous given the rest of our docs.

Reply:
Fair enough, I think `deploy` is the clearest option.

Reply:
Agreed.